_overviews/scala3-book/domain-modeling-fp.md

```diff
@@ -73,13 +71,36 @@ enum CrustType:
 enum Topping:
   case Cheese, Pepperoni, BlackOlives, GreenOlives, Onions
 ```
+
+{% endtab %}
+{% endtabs %}
+
 > Data types that describe different alternatives (like `CrustSize`) are also sometimes referred to as _sum types_.
 
 ### Describing Compound Data
 
 A pizza can be thought of as a _compound_ container of the different attributes above.
 We can use a `case` class to describe that a `Pizza` consists of a `crustSize`, `crustType`, and potentially multiple `Topping`s:
 
+{% tabs data_2 class=tabs-scala-version %}
+{% tab 'Scala 2' for=data_2 %}
+
+```scala
+import CrustSize._
```

Comment:
There needs to be a scala 2 version of these enums otherwise there is nothing to import from here

Reply:
changed to Scala3 Only.